packages/editor/src/components/rich-text/input-event.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export class RichTextInputEvent extends Component {
```

Comment:
A unit test for this component could be good.

Reply:
How would it work?

Reply:
Something like this pseudocode:

```js
const onInput = jest.fn();

const instance = mount( <RichTextInputEvent inputType="ella-is-cool" onInput={ onInput } /> );

const event = new Event( 'input' );
event.inputType = 'ella-is-cool';
document.dispatchEvent( event );

expect( onInput ).toHaveBeenCalled();
```

I find these sorts of tests are good for catching silly regressions in the future, e.g. typos.